src/components/page-about/index.js

```diff
@@ -1,15 +1,17 @@
+import { Typography } from '@equinor/eds-core-react';
 import React from 'react';
+
 import ConfigList from '../config-list';
 import Panel from '../panel';
 
 const PageAbout = () => {
   return (
     <Panel>
```

Comment:
to add spacing and maintain styling, replace <Panel> with <div className="panel grid grid--gap-small">